test/org/zaproxy/zap/extension/pscanrules/TestInfoPrivateAddressDisclosureUnitTest.java

```diff
@@ -227,6 +228,32 @@ public void alertsIfPrivIpAndDropsPortNoInEvidence() throws HttpMalformedHeaderE
 			}
 		}
 	}
+	
+	@Test
+	public void shouldNotAlertIfPrivIpIsTarget() throws HttpMalformedHeaderException, URIException {
```

Comment:
Could this test be moved closer to related tests? Or update the existing test (`shouldIgnoreRequestedPrivateIpByDefault`) that checks the same?